natlas-server/app/admin/routes.py

```diff
@@ -81,7 +82,7 @@ def delete_user(id):
         user = User.query.filter_by(id=id).first()
         User.query.filter_by(id=id).delete()
         db.session.commit()
-        flash(f"{user.email} deleted!", "success")
+        flash(f"{user.email} deleted.", "success")
```

Comment:
You could create a flash_success method that validated it ends in a "." and always passes "success".